ppb/sprites.py

```diff
@@ -127,7 +127,7 @@ def center(self, value):
         self.parent.position = self._mk_update_vector_center(value)
 
     def _mk_update_vector_side(self, attribute, value):
-        value = Vector(*value)
+        value = Vector.convert(value)
```

Comment:
`Vector(value)` or `Vector(*value)` should work perfectly fine, no?

Reply:
This was written before the copy constructor was merged.

Reply:
'k.  To be fair, `Vector.convert()` might be avoiding a copy when `value` is already a `Vector`.

Reply:
By design, yes.

Reply:
Yes, I was just saying that I just realised using `Vector.convert` is better there.
Would it be possible to avoid the copy when using `Vector()` ?

Reply:
It's conventional for the `MyType(value)` to be a copy constructor, like `list()`, `dict()`, `set()`, etc

Reply:
So, this is conventional for mutable types.

It looks like immutable types tend to not copy.